Sources/TestingMacros/Support/ConditionArgumentParsing.swift

```diff
@@ -279,15 +291,9 @@ private func _exprFromOptionalChainedExpr(_ expr: some ExprSyntaxProtocol) -> (E
   // the member accesses in the expression use optional chaining and, if one
   // does, ensure we preserve optional chaining in the macro expansion.
   if questionMarkCount == 0 {
-    func isOptionalChained(_ expr: some ExprSyntaxProtocol) -> Bool {
-      if expr.is(OptionalChainingExprSyntax.self) {
-        return true
-      } else if let memberAccessBaseExpr = expr.as(MemberAccessExprSyntax.self)?.base {
-        return isOptionalChained(memberAccessBaseExpr)
-      }
-      return false
-    }
-    if isOptionalChained(originalExpr) {
+    let optionalChainFinder = _OptionalChainFinder(viewMode: .sourceAccurate)
```

Comment:
Nit: did you consider defining a `static` helper on the new class that would allow the usage here to be a one-liner?

Reply:
That would total more lines of code than just calling it as-is. :)